src/mbgl/renderer/painter.cpp

```diff
@@ -87,12 +99,15 @@ Painter::Painter(gl::Context& context_,
       state(state_),
       tileVertexBuffer(context.createVertexBuffer(tileVertices())),
       rasterVertexBuffer(context.createVertexBuffer(rasterVertices())),
+      extrusionTextureVertexBuffer(context.createVertexBuffer(extrusionTextureVertices())),
       tileTriangleIndexBuffer(context.createIndexBuffer(tileTriangleIndices())),
-      tileBorderIndexBuffer(context.createIndexBuffer(tileLineStripIndices())) {
+      tileBorderIndexBuffer(context.createIndexBuffer(tileLineStripIndices())),
+      extrusionTextureTriangleIndexBuffer(context.createIndexBuffer(tileTriangleIndices())) {
```

Comment:
Since this contains the same data as `tileTriangleIndexBuffer`, let's reuse that buffer. We can rename to `quadTriangleIndexBuffer` / `quadTriangleIndices` for generality.